services/virtualmachine/sdk_ethereum.go

```diff
@@ -85,6 +85,7 @@ func (s *service) handleSdkEthereumCall(ctx context.Context, executionContext *e
 // inputArg4: ethereumABIPackedInputArguments ([]byte)
 // outputArg0: ethereumABIPackedOutput ([]byte)
 func (s *service) handleSdkEthereumCallMethod(ctx context.Context, executionContext *executionContext, args []*protocol.Argument, permissionScope protocol.ExecutionPermissionScope) ([]byte, error) {
+	panic("Ethereum SDK Disabled") // TODO Ethereum Disconnect
```

Comment:
this is not a TODO anymore.. you can write `// ethereum access API suspended` or something. at best its a `// TODO re-enable Ethereum access (with refTime based finality)` or something